gocourse10/feed/feed.go

```diff
@@ -0,0 +1,59 @@
+package feed
+
+import (
+	"log"
+)
+
+func Feed(c Collar) {
+	log.Printf("Feed with %s \n", c.Trough.Food())
+}
+
+type Trough interface {
+	Food() string
+}
+
+type MeetTrought struct {
+}
+
+func (MeetTrought) Food() string {
+	return "Ham, pork and chicken"
+}
+
+type VegetablesTrought struct {
```

Comment:
https://www.merriam-webster.com/dictionary/trough
```suggestion
type VegetablesTrough struct {
```